pkg/controller/storagecluster/storagecluster.go

```diff
@@ -308,6 +310,22 @@ func (c *Controller) RegisterCRD() error {
 	return nil
 }
 
+func (c *Controller) isDaemonSetPresent(name string) (bool, error) {
```

Comment:
So far, we have maintained all portworx specific code under drivers/storage/portworx and the storagecluster controller only contains generic logic. It would be nice to continue and have that separation.
Can we create a Validate() function under drivers/storage.Driver interface and then implement this logic in the portworx.go file? From storagecluster we just call Driver.Validate() and return the error.